ccm_web/package.json

```diff
@@ -3,19 +3,28 @@
   "version": "1.0.0",
   "description": "",
   "scripts": {
-    "dev:server": "nodemon --inspect=0.0.0.0:9229 --config nmon_config.json server/app.ts",
+    "dev:server": "nodemon --inspect=0.0.0.0:9229 --config nmon_config.json server/src/main.ts",
     "dev:client": "webpack --watch --config webpack/webpack.dev.ts",
     "dev": "npm run dev:server & npm run dev:client",
-    "build-client": "webpack --config webpack/webpack.prod.ts",
-    "build-server": "tsc",
-    "preprod": "npm run build-client && npm run build-server",
-    "prod": "NODE_ENV=production node dist/server/app.js"
+    "build:client": "webpack --config webpack/webpack.prod.ts",
+    "build:server": "tsc --project tsconfig.build.json",
+    "preprod": "npm run build:client && npm run build:server",
+    "prod": "NODE_ENV=production node dist/server/src/main.js",
```

Comment:
Running in Prod mode with `docker-compose-prod.yml` is throwing me this error. I don't see this issue in Dev mode

```
ccm_web_prod | internal/modules/cjs/loader.js:883
ccm_web_prod |   throw err;
ccm_web_prod |   ^
ccm_web_prod |
ccm_web_prod | Error: Cannot find module '/app/server/app.js'
ccm_web_prod |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
ccm_web_prod |     at Function.Module._load (internal/modules/cjs/loader.js:725:27)
ccm_web_prod |     at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
ccm_web_prod |     at internal/main/run_main_module.js:17:47 {
ccm_web_prod |   code: 'MODULE_NOT_FOUND',
ccm_web_prod |   requireStack: []
ccm_web_prod | }
```

Reply:
Ah, yeah, I need to update the entrypoint in `Dockerfile`.

Reply:
This should be okay now.